src/main/java/address/controller/MainController.java

```diff
@@ -435,6 +437,29 @@ public void showTagList(ObservableList<Tag> tags) {
     }
 
     /**
+     * Opens a dialog to show the help page
+     */
+    public void showHelpPage() {
+        logger.debug("Loading help page.");
+        final String fxmlResourcePath = FXML_HELP;
+        try {
+            // Load the fxml file and create a new stage for the popup dialog.
+            FXMLLoader loader = loadFxml(fxmlResourcePath);
+            AnchorPane page = loader.load();
+
+            Scene scene = new Scene(page);
+            Stage dialogStage = loadDialogStage("Help", null, scene);
+            dialogStage.getIcons().add(getImage(ICON_HELP));
+            // Show the dialog and wait until the user closes it
+            dialogStage.showAndWait();
+        } catch (IOException e) {
+            logger.fatal("Error loading tag list view: {}", e);
+            showFatalErrorDialogAndShutdown("FXML Load Error", "Cannot load fxml for tag list.",
+                    "IOException when trying to load ", fxmlResourcePath);
+        }
```

Comment:
Two things to improve:
1. try to avoid very tall `try` blocks (or any blocks). Most likely it can be narrowed to the first few lines?
2. This try-catch structure seems to be repetitive. Any way to abstract it out?